pkg/eventprocessor/eventfeed/impl/eventfeed.go

```diff
@@ -163,6 +163,7 @@ func (ef *EventFeed) Start(
 				// in the next head. Probably the API can have transient unavailability.
 				ef.log.Warn().Err(err).Msgf("filter logs from %d to %d", fromHeight, toHeight)
 				if strings.Contains(err.Error(), "read limit exceeded") ||
+					strings.Contains(err.Error(), "Log response size exceeded") ||
```

Comment:
While testing the PR locally, I noticed that Arbitrum use a slightly different error message that we weren't detecting. 

I dug if this was happening in the cloud validator, and wasn't the case. I think this is because the cloud validator was always syncing this chain from new blocks, but in my case was lagged behind and thus syncing events in bigger block ranges which can trigger this error.

In any case, we detect it and do the usual shrinking of block ranges. This fixed the problem in my local node as expected.